# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../less/utf8-detection.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

I have no locales with UTF-8 or utf-8 ...

  - Rene Rebe <rene@exactcode.de>

--- less-382/charset.c.vanilla	2005-03-24 01:15:59.000000000 +0100
+++ less-382/charset.c	2005-03-24 01:18:57.000000000 +0100
@@ -251,7 +251,8 @@
 	    (s = lgetenv("LC_CTYPE")) != NULL ||
 	    (s = lgetenv("LANG")) != NULL)
 	{
-		if (strstr(s, "UTF-8") != NULL || strstr(s, "utf-8") != NULL)
+		if (strstr(s, "UTF8") != NULL || strstr(s, "utf8") != NULL ||
+		    strstr(s, "UTF-8") != NULL || strstr(s, "utf-8") != NULL)
 			if (icharset("utf-8"))
 				return;
 	}
